tools/releasable/Sakefile

```diff
@@ -23,8 +23,8 @@ use File::Directory::Tree;
 my $THREADS  = 7;
 
 # These are used so that you can run tests on a faster machine.
-my $SSH      = ‘alex@localhost’; # TODO make it configurable
-my $SSH-PATH = ‘/home/alex/git/Releasable/’; # TODO make it configurable
+my $SSH      = %*ENV<RELEASE_SSH> // "{qx{whoami}.trim}@localhost";
+my $SSH-PATH = %*ENV<RELEASE_HOME>;
```

Comment:
This might work, but I have a feeling that I already had it like this and it didn't. Once it ssh's into a different machine, the ENV won't be propagated. Can you test it that it works when using two different machines?

Reply:
Sadly, I can't as I have only a single laptop available around.
Also, if this is bad, I'd rather looked into making the whole ssh layer optional because it, in my opinion, ought to be so, but decided to remove some hardcoded values firstly.